src/components/controls/Dropdown.tsx

```diff
@@ -17,67 +17,42 @@ class Dropdown extends React.Component
     constructor(props: any) {
         super(props);
         this.state = {
-            isOpen: false,
+            selectedName: undefined,
+            selectedValue: undefined,
```

Comment:
Why are they undefined when selected is a mandatory prop.

Reply:
If it is not undefined, if I set it as '', there is a warning at console:
Warning: Select elements must be either controlled or uncontrolled (specify either the value prop, or the defaultValue prop, but not both). Decide between using a controlled or uncontrolled select element and remove one of these props

Reply:
Could it not be set as `props.selected` ?